com.unicornsonlsd.finamp.yaml

```diff
@@ -0,0 +1,299 @@
+app-id: com.unicornsonlsd.finamp
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: finamp
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --share=network
+modules:
+  - name: finamp
+    buildsystem: simple
+    build-commands:
+      - cp -a bundle/. /app/
+      - chmod +x /app/finamp
+      - mkdir -p /app/bin
+      - ln -s /app/finamp /app/bin/finamp
+      - install -Dm644 icons/16x16/apps/finamp.png /app/share/icons/hicolor/16x16/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/32x32/apps/finamp.png /app/share/icons/hicolor/32x32/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/48x48/apps/finamp.png /app/share/icons/hicolor/48x48/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/64x64/apps/finamp.png /app/share/icons/hicolor/64x64/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/128x128/apps/finamp.png /app/share/icons/hicolor/128x128/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/256x256/apps/finamp.png /app/share/icons/hicolor/256x256/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/384x384/apps/finamp.png /app/share/icons/hicolor/384x384/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/512x512/apps/finamp.png /app/share/icons/hicolor/512x512/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icon_foreground.svg /app/share/icons/hicolor/scalable/apps/com.unicornsonlsd.finamp.svg
+      - m4 finamp.desktop.m4 > com.unicornsonlsd.finamp.desktop
```

Comment:
this kind of shenanigans should really be part of the upstream build system as it should really output a valid .desktop. .desktop are not flatpak specific and have been around 20+ years.